README.md

```diff
@@ -70,10 +71,16 @@ platforms:
   - name: small
     driver:
       catalog_id: 8a189191-fea6-43eb-981e-ee0fa40f8f57
+      extra_parameters:
+        provider-mycustompropname: {type: "string",value: "smallvalue"}
```

Comment:
While this notation is technically correct and will work, the more "proper" way to notate nested hashes in YAML is like this:

```yaml
extra_parameters:
  provider-mycustompropname:
    type: string
    value: smallvalue
  provider-Vrm.DataCenter.Location:
    type: string
    value: Non-Prod
```

And you don't need to quote strings in YAML :)